### PR TITLE
Kandji C2C source doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create Kandji C2C source.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Create Kandji C2C Public doc](https://sumologic.atlassian.net/browse/DOCS-486)

### CODE REVIEW

File: blog-service/2024-10-11-collection.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - collection
+  - kandji
+hide_table_of_contents: true    
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+We're excited to announce the release of our new cloud-to-cloud source for Kandji. This source helps you to collect threat details, devices list, device activities, and device details from the Kandji platform and ingests it into Sumo Logic. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kandji-source).
```
Comment (kimsauce) [inline blog-service/2024-10-11-collection.md:14]:
```suggestion
We're excited to announce the release of our new cloud-to-cloud source for Kandji. This source helps you to collect threat details, device lists, activity logs, and device information from the Kandji platform, and ingest them into Sumo Logic for streamlined analysis. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kandji-source).
```

File: static/files/c2c/kandji/example.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "pollingIntervalHour": 12,
+  "collectDeviceActivities": true,
+  "collectDeviceDetails": true
+},
+		"schemaRef": {
+            		"type": "Kandji"
+        	},
+        	"sourceType": "Universal"
+	}
+}
```
Comment (kimsauce) [inline static/files/c2c/kandji/example.json:20]:
```suggestion
{
	"api.version":"v1",
	"source":{
		"config":{
			"name":"Kandji",
			"requestEndpoint":"https://sumo.api.kandji.io",
			"bearerToken":"xxxxxxxxxxxxxxxxxxxxxxxx",
			"collectThreatDetails":true,
			"collectDevices":true,
			"pollingIntervalMin":5,
			"pollingIntervalHour":12,
			"collectDeviceActivities":true,
			"collectDeviceDetails":true
		},
		"schemaRef":{
			"type":"Kandji"
		},
		"sourceType":"Universal"
	}
}
```

Comment (kimsauce) [inline static/files/c2c/kandji/example.json:20]:
```suggestion
{
  "api.version":"v1",
  "source":{
    "config":{
      "name":"Kandji",
      "requestEndpoint":"https://sumo.api.kandji.io",
      "bearerToken":"xxxxxxxxxxxxxxxxxxxxxxxx",
      "collectThreatDetails":true,
      "collectDevices":true,
      "pollingIntervalMin":5,
      "pollingIntervalHour":12,
      "collectDeviceActivities":true,
      "collectDeviceDetails":true
    },
    "schemaRef":{
      "type":"Kandji"
    },
    "sourceType":"Universal"
  }
}
```

**Review by kimsauce — APPROVED** [review]

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kandji-source.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. In **Endpoint URL**, enter the endpoint URL collected from the Kandji platform.
+1. In **Bearer Token**, enter the bearer token collected from the Kandji platform.
+1. Select the **Collect Threat Details** checkbox to collect threat data. By default, **Collect Threat Details** checkbox will be selected.
+1. The **Threat Details Interval** is set for 5 minutes hours by default. You can adjust it based on your needs.
+1. In the Devices section, select the type of devices data: **Collect Devices**, **Collect Device Activities**, and/or **Collect Device Details**.
+1. The **Devices Interval** is set for 12 hours hours by default. You can adjust it based on your needs.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kandji-source.md:61]:
```suggestion
1. The **Devices Interval** is set for 12 hours hours by default. You can adjust it based on your needs.
1. **Processing Rules**. Configure any desired filters, such as allowlist, deny list, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kandji-source.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+1. (Optional) **Fields**. Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. In **Endpoint URL**, enter the endpoint URL collected from the Kandji platform.
+1. In **Bearer Token**, enter the bearer token collected from the Kandji platform.
+1. Select the **Collect Threat Details** checkbox to collect threat data. By default, **Collect Threat Details** checkbox will be selected.
+1. The **Threat Details Interval** is set for 5 minutes hours by default. You can adjust it based on your needs.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/kandji-source.md:59]:
```suggestion
1. The **Threat Details Interval** is set for 5 minutes by default. You can adjust it based on your needs.
```

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@JV0812, this PR was merged by @yleiferman because Kandji was released today.

**Pull request merged**